src/domains/observability/env/prod/terraform.tfvars

```diff
@@ -259,13 +259,31 @@ ehns_metric_alerts = {
       }
     ],
   },
+  no_wallet_ingestion_alert = {
+    aggregation = "Total"
+    metric_name = "IncomingMessages"
+    description = "Payment wallet onboarding written events less than 2000 detected in the last 24h"
+    operator    = "LessThanOrEqual"
+    threshold   = 2000
```

Comment:
This threshold is too low; it is very close to the low-traffic onboarding number (see the traffic from November  2)

Reply:
Yes, maybe we can lower to 1000 in one day and see how it will go in prod after activating it

Reply:
Lowered to 1000 with https://github.com/pagopa/pagopa-infra/pull/2608/commits/23ab4ea37e9438c30995ed3a22cd5a59d8a690ec